app/src/main/java/com/keylesspalace/tusky/entity/Status.kt

```diff
@@ -42,7 +42,7 @@ data class Status(
     val visibility: Visibility,
     @SerializedName("media_attachments") var attachments: ArrayList<Attachment>,
     val mentions: List<Mention>,
-    val tags: List<HashTag>,
+    val tags: List<HashTag>?,
```

Comment:
I think it is better to have the tags nullable, because there will already be statuses in the database and if we assume they have no tags they will not open in Tusky.
(empty List vs null, see here https://github.com/tuskyapp/Tusky/blob/develop/app/src/main/java/com/keylesspalace/tusky/util/LinkHelper.kt#L103-L104 )